R/geom-label.R

```diff
@@ -67,6 +67,7 @@ GeomLabel <- ggproto("GeomLabel", Geom,
     }
 
     data <- coord$transform(data, panel_params)
+
```

Comment:
Can you avoid touching files and functions that you don't change

Reply:
Sorry, this was an accident. Left from from en edit that I reverted during development. Will fix in a minute.